lib/components/viewers/nearby/nearby-view.tsx

```diff
@@ -224,9 +238,19 @@ function NearbyView({
         .flat(Infinity)
     )
   )
+
+  // If configured, filter out stops that don't have any patterns
+  const filteredNearby = nearby?.filter((n: any) => {
+    if (n.place.__typename === 'Stop' && nearbyViewConfig?.hideEmptyStops) {
+      const patternArray = patternArrayforStops(n.place, routeSortComparator)
```

Comment:
rename to `patternArrayForStops`

Reply:
Unfortunately that's going to conflict with the `patternArrayforStops` being imported. I agree it would be cleaner though!